client/controller/models/apps/apps.go

```diff
@@ -97,15 +97,12 @@ func Logs(c *client.Client, appID string, lines int) (string, error) {
 
 	body, err := c.BasicRequest("GET", u, nil)
 
-	if err != nil {
-		return fmt.Sprintf("Error:%v", err), err
```

Comment:
Got it, thanks @jchauncey. We should roll something like this out for more commands in another PR. See #356